### PR TITLE
fix: launch review reminder

### PR DESCRIPTION
**Pull request by Xazin**
### Feature Preview

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart [hunk 1]
```diff
@@ -25,9 +27,17 @@ class MobileGridDateCellSkin extends IEditableDateCellSkin {
...
-            fontSize: 15,
+          child: Row(
+            children: [
+              if (state.data?.reminderId.isNotEmpty ?? false) ...[
+                const FlowySvg(FlowySvgs.clock_alarm_s),
+                const HSpace(6),
+              ],
+              FlowyText(
+                state.dateStr,
+                fontSize: 15,
```
Comment (richardshiue) [inline frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart:38]: I missed handling the overflow here. Can you fix it for me?

Comment (Xazin) [reply frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart:38]: Will do

Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart:38]: Oh nvm. It's in a singlechildscrollview

Comment (Xazin) [reply frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_date_cell.dart:38]: Even better

**Review by richardshiue — APPROVED** [review]
Barring the translation, LGTM

**Pull request merged**